BCStudentSoftwareDevTeam/celts
# Renew/Reschedule Canceled Events Issue 986

Issue: 
There was a request to have a button to be able to renew/reschedule a canceled event.

Fix:
Added modal which allows supervisors to reschedule previously canceled events. We take data from the page of the canceled event and send that data to a route which uses the current event's information to copy the event contents over.

Test:
1. Reset database
2. Create a new event from the create event option in the sidebar
3. Add RSVP and waitlister students from the track volunteers page
4. cancel the event from the event view page
5. Try different dates to see if the new event reflects changes and ensure rsvps were properly copied to new event

Fixes #986
